src/http-client.ts

```diff
@@ -75,6 +76,10 @@ export class HttpClient {
         let status = response.status;
         let correlationId = response.headers?.[correlationIdHeaderName];
         let body = response.data;
+        let error: IAxiosResponseError = {
+          ...res,
+          text: response.request.response
```

Comment:
what's the difference between `response.response` vs `response.request.response`?

Reply:
`response.response` doesn't exist on `AxiosError`